### PR TITLE
Allow initial camera position to be customized before map loads

### PR DESCRIPTION
**Pull request by avakkachen**
Allow initial camera position to be customized before map loads

### CODE REVIEW

File: Samples/java/src/main/java/com/phunware/java/sample/BluedotLocationActivity.java [hunk 1]
```diff
@@ -26,6 +26,8 @@ a copy of this software and associated documentation files (the
 other dealings in this Software without prior written authorization
 from Phunware, Inc. */
 
+import static com.phunware.java.sample.CustomMapFragment.getInstance;
```
Comment (npike) [inline Samples/java/src/main/java/com/phunware/java/sample/BluedotLocationActivity.java]: Nitpick: Static import of the `getInstance()` method makes the code a little hard to read when we are doing our fragment transaction, as it's not immediately apparent on line 84 as to what fragment we are replacing in the layout.

File: Samples/java/src/main/java/com/phunware/java/sample/CustomMapFragment.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import com.phunware.mapping.MapFragment;
+import com.phunware.mapping.OnPhunwareMapReadyCallback;
+
+/**
+ * Example of extending {@link MapFragment} to customise
+ * the initial camera location and the zooming level of the map.
+ */
+public class CustomMapFragment extends MapFragment {
+
+    public static MapFragment getInstance(AppCompatActivity activity, OnPhunwareMapReadyCallback callback) {
```
Comment (npike) [inline Samples/java/src/main/java/com/phunware/java/sample/CustomMapFragment.java]: Should this return `CustomMapFragment` like the kotlin sample?

**Comment by npike** [comment]
Should we merge this into `sample_code_updates` instead of `develop`?

File: Samples/java/src/main/java/com/phunware/java/sample/CustomMapFragment.java (hunk 2 above)
Comment (avakkachen) [reply Samples/java/src/main/java/com/phunware/java/sample/CustomMapFragment.java]: We can do that.
 I'm ok with merging this to sample_code_updates. 
This is probably based off of master. Now that we're having all the code in sample_code_updates, it'll makes sense to add it there

**Pull request closed without merging**